### ISSUE TITLE
ReviewActivity is barely visible in Landscape Mode

### ISSUE THREAD

**Issue by Ayan-10** [commons-app/apps-android-commons#4343]
**Summary:** 

Elements of ReviewActivity is not visible to user in landscape. We should make a landscape layout for it.

**Steps to reproduce:** 

1.  Go to ReviewActivity
2.  Turn on landscape Mode

**What did you expect the app to do, and what did you see instead?**

Image in ReviewActivity is not visible in Landscape Mode even the text also is not visible. ReviewActivity should visible properly in landscape mode also. 

**Device and Android version:** 

Pixel 4 API 27
 
**Commons app version:** 

3.0.0 latest master

**Screen-shots:** 

<img src="https://user-images.githubusercontent.com/71203077/114088885-23bc5a80-98d3-11eb-8f04-150bc00a1d25.png" width="500">

**Would you like to work on the issue?**

Yes

**Comment by madhurgupta10** [comment]
@Ayan-10 I agree, we should have a different UI for landscape mode. Pinging @neslihanturan @nicolas-raoul @misaochan for their feedback on this :)

**Comment by misaochan** [comment]
I agree. Maybe picture on the left and text on the right?

**Comment by Ayan-10** [comment]
@madhurgupta10 @misaochan Can you once go through Nearby Activity in Landscape mode. I think the map view is becoming very tiny in landscape mode.

![Screenshot_20210428_162111](https://user-images.githubusercontent.com/71203077/116392824-89e72e00-a83e-11eb-960c-e03a93291fe9.png)

**Comment by Ayan-10** [comment]
> I agree. Maybe picture on the left and text on the right?

@madhurgupta10 @misaochan I did this for ReviewActivity. Please have a look.

![Screenshot_20210428_165048](https://user-images.githubusercontent.com/71203077/116395723-1e06c480-a842-11eb-99f3-d47a3ec82f90.png)

**Comment by madhurgupta10** [comment]
> @madhurgupta10 @misaochan Can you once go through Nearby Activity in Landscape mode. I think the map view is becoming very tiny in landscape mode.
> 
> ![Screenshot_20210428_162111](https://user-images.githubusercontent.com/71203077/116392824-89e72e00-a83e-11eb-960c-e03a93291fe9.png)

You can make a separate issue for other Activities

**Comment by madhurgupta10** [comment]
> > I agree. Maybe picture on the left and text on the right?
> 
> @madhurgupta10 @misaochan I did this for ReviewActivity. Please have a look.
> 
> ![Screenshot_20210428_165048](https://user-images.githubusercontent.com/71203077/116395723-1e06c480-a842-11eb-99f3-d47a3ec82f90.png)

Looks good to me! Let's wait for other's feedback

**Comment by misaochan** [comment]
LGTM as well. :)

**Comment by nicolas-raoul** [comment]
I am not sure how many people will do reviews in landscape mode, but if it's not too much work how about:
- Making the legend (the text that starts with `File:Mark of Zorro`...) a bit higher. It is 2 lines and I believe that the blue bottom banner (with the 4 dots) is covering the second line.
- Maybe putting the `Skip this image` part on the right only? That will leave more space for the picture.

Thanks!

**Comment by Prince-kushwaha** [comment]
@madhurgupta10 @nicolas-raoul can i work on this

**Comment by madhurgupta10** [comment]
> @madhurgupta10 @nicolas-raoul can i work on this

@Ayan-10 can you confirm if you are still interested to work on this? If not we can assign it to you @Prince-kushwaha

**Comment by Ayan-10** [comment]
@madhurgupta10 Yes, I am currently working on it. Actually, I am almost done with the UI changes as proposed by @nicolas-raoul . But I stuck a little at a point for making changes but I am trying my best to fix it. I will make PR as soon as possible.

**Comment by madhurgupta10** [comment]
@Ayan-10 Thanks for the update, feel free to ask if you need any help. @Prince-kushwaha you can work on other issues in that case.

**Comment by Prince-kushwaha** [comment]
@madhurgupta10 can i create issue for this and work on it https://github.com/commons-app/apps-android-commons/issues/4343#issuecomment-828974126

**Comment by madhurgupta10** [comment]
> @madhurgupta10 can i create issue for this and work on it [#4343 (comment)](https://github.com/commons-app/apps-android-commons/issues/4343#issuecomment-828974126)

Yes but before starting working on it, do discuss your approach and UI

**Comment by Ayan-10** [comment]
> @Ayan-10 Thanks for the update, feel free to ask if you need any help. @Prince-kushwaha you can work on other issues in that case.

Thanks, @madhurgupta10 .



I solved the issue and this is the final UI. Please check out @madhurgupta10 @nicolas-raoul @misaochan .

![Screenshot_20210513_201557](https://user-images.githubusercontent.com/71203077/118170473-c2465900-b447-11eb-932d-059685514f90.png)

Should I make PR with this UI?

**Comment by madhurgupta10** [comment]
> > @Ayan-10 Thanks for the update, feel free to ask if you need any help. @Prince-kushwaha you can work on other issues in that case.
> 
> Thanks, @madhurgupta10 .
> 
> I solved the issue and this is the final UI. Please check out @madhurgupta10 @nicolas-raoul @misaochan .
> 
> ![Screenshot_20210513_201557](https://user-images.githubusercontent.com/71203077/118170473-c2465900-b447-11eb-932d-059685514f90.png)
> 
> Should I make PR with this UI?

Please make a PR, thanks!
